ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -949,6 +1260,8 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = A97451351B3A935E00F48E55 /* watchOS-Framework.xcconfig */;
 			buildSettings = {
+				CLANG_ENABLE_MODULES = YES;
+				DEFINES_MODULE = YES;
```

Comment:
Done in https://github.com/ReactiveCocoa/ReactiveCocoa/pull/3210/commits/23ae7c002fa2b3c05fb04797ed0457c0c8e05ee4.